python/spec/dependabot/python/file_parser_spec.rb

```diff
@@ -65,6 +65,29 @@
       its(:length) { is_expected.to eq(3) }
     end
 
+    context "with jinja templates" do
+      let(:requirements_fixture_name) { "jinja_requirements.txt" }
```

Comment:
Nit, usually add a newline between `let`s and `describe/context`
```suggestion
      let(:requirements_fixture_name) { "jinja_requirements.txt" }

```